Bytes handled incorrectly in JDBC Sink for Postgres

In [PostgreSqlDialect.java](https://github.com/confluentinc/kafka-connect-jdbc/blob/master/src/main/java/io/confluent/connect/jdbc/sink/dialect/PostgreSqlDialect.java), the Schema.Type `BYTES` is mapped to `BLOB`, which [does not exist in Postgres](https://www.postgresql.org/docs/10/static/datatype-binary.html). Instead, it should be mapped to `BYTEA`.

Did this issue ever get resolved and released?  I'm experiencing it in version kafka-connect-jdbc: 5.0.1
[2018-11-22 10:48:51,202] ERROR WorkerSinkTask{id=jdbc-sink-0} RetriableException from SinkTask: (org.apache.kafka.connect.runtime.WorkerSinkTask:577)
org.apache.kafka.connect.errors.RetriableException: java.sql.SQLException: org.postgresql.util.PSQLException: ERROR: type "blob" does not exist
  Position: 246

        at io.confluent.connect.jdbc.sink.JdbcSinkTask.put(JdbcSinkTask.java:94)
        at org.apache.kafka.connect.runtime.WorkerSinkTask.deliverMessages(WorkerSinkTask.java:564)
        at org.apache.kafka.connect.runtime.WorkerSinkTask.poll(WorkerSinkTask.java:322)
        at org.apache.kafka.connect.runtime.WorkerSinkTask.iteration(WorkerSinkTask.java:225)
        at org.apache.kafka.connect.runtime.WorkerSinkTask.execute(WorkerSinkTask.java:193)
        at org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:175)
        at org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:219)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.sql.SQLException: org.postgresql.util.PSQLException: ERROR: type "blob" does not exist

--- a/src/main/java/io/confluent/connect/jdbc/dialect/PostgreSqlDatabaseDialect.java
+++ b/src/main/java/io/confluent/connect/jdbc/dialect/PostgreSqlDatabaseDialect.java
@@ -211,7 +211,7 @@ public class PostgreSqlDatabaseDialect extends GenericDatabaseDialect {
       case STRING:
         return "TEXT";
       case BYTES:
-        return "BLOB";
+        return "BYTEA";
       default:
         return super.getSqlType(field);
     }

--- a/src/test/java/io/confluent/connect/jdbc/dialect/PostgreSqlDatabaseDialectTest.java
+++ b/src/test/java/io/confluent/connect/jdbc/dialect/PostgreSqlDatabaseDialectTest.java
@@ -44,7 +44,7 @@ public class PostgreSqlDatabaseDialectTest extends BaseDialectTest<PostgreSqlDat
     assertPrimitiveMapping(Type.FLOAT32, "REAL");
     assertPrimitiveMapping(Type.FLOAT64, "DOUBLE PRECISION");
     assertPrimitiveMapping(Type.BOOLEAN, "BOOLEAN");
-    assertPrimitiveMapping(Type.BYTES, "BLOB");
+    assertPrimitiveMapping(Type.BYTES, "BYTEA");
     assertPrimitiveMapping(Type.STRING, "TEXT");
   }
 
@@ -66,7 +66,7 @@ public class PostgreSqlDatabaseDialectTest extends BaseDialectTest<PostgreSqlDat
     verifyDataTypeMapping("DOUBLE PRECISION", Schema.FLOAT64_SCHEMA);
     verifyDataTypeMapping("BOOLEAN", Schema.BOOLEAN_SCHEMA);
     verifyDataTypeMapping("TEXT", Schema.STRING_SCHEMA);
-    verifyDataTypeMapping("BLOB", Schema.BYTES_SCHEMA);
+    verifyDataTypeMapping("BYTEA", Schema.BYTES_SCHEMA);
     verifyDataTypeMapping("DECIMAL", Decimal.schema(0));
     verifyDataTypeMapping("DATE", Date.SCHEMA);
     verifyDataTypeMapping("TIME", Time.SCHEMA);

